apps/dashboard/src/modules/user/components/UserSettingsComponent.vue

```diff
@@ -89,16 +89,38 @@ import apiService from "@/services/ApiService";
 import { handleError } from "@/utils/errorUtils";
 
 async function startScan() {
-  if (!('NDEFReader' in window)) {
+
+  const isIOS = /iPhone|iPad|iPod/i.test(navigator.userAgent);
+  const isAndroid = /Android/i.test(navigator.userAgent);
+
+  if (!('NDEFReader' in window) && !(isIOS || isAndroid)) {
     toast.add({
-          severity: "error",
-          summary: t("common.toast.error.error"),
-          detail: t("common.toast.error.unsupported"),
-          life: 3000,
-        });
+      severity: "error",
+      summary: t("common.toast.error.error"),
+      detail: t("common.toast.error.desktopUnsupported"),
+      life: 3000,
+    });
     return;
   }
 
+  if (!('NDEFReader' in window) && isAndroid) {
+    toast.add({
+      severity: "error",
+      summary: t("common.toast.error.error"),
+      detail: t("common.toast.error.androidUnsupported"),
+      life: 3000,
+    });
+  }
+
+  if (!('NDEFReader' in window) && isIOS) {
+    toast.add({
+      severity: "error",
+      summary: t("common.toast.error.error"),
+      detail: t("common.toast.error.iosUnsupported"),
+      life: 3000,
+    });
+  }
+
```

Comment:
can be simplified by having a single branch to check if the NDEFReader is not in the window and then simply switch case what the detail should be.